ui/page/overlay/index.js

```diff
@@ -0,0 +1,4 @@
+import { connect } from 'react-redux';
+import OverlayPage from './view';
+
+export default connect()(OverlayPage);
```

Comment:
No need to `connect` if not using redux.  just import and export back directly.

Reply:
Not relevant once we moved this outside
https://github.com/OdyseeTeam/odysee-frontend/pull/348#discussion_r758904062